Example/SmileID/HomeView.swift

```diff
@@ -80,20 +99,27 @@ struct HomeView: View {
                     .font(SmileID.theme.body)
                     .foregroundColor(SmileID.theme.onLight)
             }
-                .toast(isPresented: $viewModel.showToast) {
-                    Text(viewModel.toastMessage)
-                        .font(SmileID.theme.body)
-                        .multilineTextAlignment(.center)
-                        .padding()
-                }
-                .padding()
-                .navigationBarTitle(Text("Smile ID"), displayMode: .inline)
-                .navigationBarItems(trailing: SmileEnvironmentToggleButton())
-                .background(SmileID.theme.backgroundLight.ignoresSafeArea())
+            .toast(isPresented: $viewModel.showToast) {
+                Text(viewModel.toastMessage)
+                    .font(SmileID.theme.body)
+                    .multilineTextAlignment(.center)
+                    .padding()
+            }
+            .padding()
+            .navigationBarTitle(Text("Smile ID"), displayMode: .inline)
+            .navigationBarItems(trailing: SmileEnvironmentToggleButton())
+            .background(SmileID.theme.backgroundLight.ignoresSafeArea())
         }
     }
 }
 
+extension HomeView {
+    func showNetworkError() {
+        viewModel.toastMessage = "No internet connection"
```

Comment:
We should be using unidirectional data flow (see [here](https://developer.android.com/jetpack/compose/architecture#udf)). That is to say, the UI should only be responsible for handling events and the UI should never update the UI itself. The viewmodel, upon receiving events, should decide what to do with them (i.e. show a toast).

Can you move this to the viewModel? 
The networkMonitor should be a member of the viewModel, not the view